### PR TITLE
Added Pearson correlation coefficient and the ability to break semeva…

### PR DESCRIPTION
**Pull request by jlowryduda**
…l results by language.

It seems to me like it's not necessary to define all of the eval names ahead of time in EVALS variable (wordsim.py). Is that a correct?

### CODE REVIEW

**Comment by rspeer** [comment]
In fact, the `EVALS` list is unnecessary now and you can remove it. (It used to be used to make the `results_in_context` dataframe, but now it just gets the keys from `COMPARISONS`, and incidentally some of that needs to be updated if I ever want to use it again.)

File: conceptnet5/vectors/evaluation/wordsim.py [hunk 1]
```diff
@@ -440,8 +518,43 @@ def evaluate_raw(frame, subset='dev'):
...
+        semeval_crosslingual = evaluate_semeval_crosslingual_global(frame)
+        results.loc['semeval17-2a'] = semeval_monolingual
+        results.loc['semeval17-2b'] = semeval_crosslingual
+    else:
+        semeval_en = evaluate_semeval_monolingual(frame, 'en')
+        semeval_de = evaluate_semeval_monolingual(frame, 'de')
+        semeval_es = evaluate_semeval_monolingual(frame, 'es')
+        semeval_it = evaluate_semeval_monolingual(frame, 'it')
+        semeval_fa = evaluate_semeval_monolingual(frame, 'fa')
+        semeval_en_de = evaluate_semeval_crosslingual(frame, 'en', 'de')
```
Comment (alin-luminoso) [inline conceptnet5/vectors/evaluation/wordsim.py]: (drive-by: if you want to be fancy, https://docs.python.org/3/library/itertools.html#itertools.combinations)
